Sage/carbon
# test(dismissible-box): migrate tests to playwright - FE-6114

### Proposed behaviour

- Add command to clear playwright cache
- Refactor cypress tests to playwright

### Current behaviour

- No command to clear playwright cache
- Tests are currently using cypress

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

- Run the new command `clear-playwright-cache` as `npm run clear-playwright-cache` and it should delete the playwright cache. Cache should reappear when the playwright tests have been ran again.
